armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -530,7 +530,7 @@ After=lightningd.service
 Type=simple
 EnvironmentFile=/etc/base-middleware/base-middleware.conf
 EnvironmentFile=/mnt/ssd/bitcoin/.bitcoin/.cookie.env
-ExecStart=/usr/local/sbin/base-middleware -rpcuser=${BITCOIN_RPCUSER} -rpcpassword=${RPCPASSWORD} -rpcport=${BITCOIN_RPCPORT} -lightning-rpc-path=${LIGHTNING_RPCPATH}
+ExecStart=/usr/local/sbin/base-middleware -rpcuser=${BITCOIN_RPCUSER} -rpcpassword=${RPCPASSWORD} -rpcport=${BITCOIN_RPCPORT} -lightning-rpc-path=${LIGHTNING_RPCPATH} -noise-key-path=/mnt/ssd/middleware
```

Comment:
@Stadicus is giving the middleware a directory here fine by you?

Reply:
Maybe just specify a more generic `datadir` similar to bitcoin, where the keyfiles and other data is expected to reside?

Midterm I could see the following two options (not for this PR):
* `conf`: specifies the configuration file, e.g. `--conf=/etc/middleware/middleware.conf`
* `datadir`: specifies the data directory, e.g. `--datadir=/mnt/ssd/system/middleware`
The `datadir` could then also be specified in the config file, so that this is the only argument we need to pass.

Reply:
Ok, systemd starts the middleware now with `-datadir=/mnt/ssd/system/middleware`

Reply:
ACK